lib/include/libYulAST/YulFunctionCallNode.h

```diff
@@ -18,7 +18,7 @@ class YulFunctionCallNode : public YulExpressionNode {
   virtual std::string to_string() override;
   YulFunctionCallNode(const json *rawAST);
   std::string getCalleeName();
-  void setCalleeName(llvm::StringRef);
+  void setCalleeName(std::string);
```

Comment:
This will copy the string. Can use C++17 string view as a drop-in replacement:
```suggestion
  void setCalleeName(std::string_view);
```